src/Movies.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Movie from './Movie';
+
+
+class Movies extends React.Component {
+    constructor() {
+        super();
+        this.watchListReceiver = this.watchListReceiver.bind(this);
+    }
+
+    watchListReceiver(title, action) {
+        this.props.receiver(title, action);
+    }
+
+    render() {
+
+        console.log(this.props.movies.length);
+        console.log(this.props.movies);
+        return (
+            this.props.movies.length > 0
```

Comment:
Avoid having multi-line ternaries in JSX. I would use an if/else and return from each block.